spec/lib/danger/request_sources/gitlab_spec.rb

```diff
@@ -400,7 +400,7 @@
 
           subject.fetch_details
 
-          subject.update_pull_request!(warnings: [v], errors: [], messages: [])
+          expect { subject.update_pull_request!(warnings: [v], errors: [], messages: []) }.not_to output(/Please convert ObjectifiedHash object/).to_stderr
```

Comment:
meh - not ideal to test here, but there weren't specific tests for the methods this one hits, so this will ensure it doesn't happen again...at least in this specific call

Reply:
heh, seems reasonable enough